template_rpm/09_cleanup.sh

```diff
@@ -49,4 +49,9 @@ fi
 
 truncate --no-create --size=0 "${INSTALL_DIR}"/var/log/dnf.*
 
+if [[ "$DIST_NAME" = 'fedora' ]] && [[ "$DIST_VER" -ge 37 ]] &&
+    rpm --root "$INSTALL_DIR" -qv qubes-core-agent-selinux; then
```

Comment:
I'd recommend `chroot "$INSTALL_DIR" rpm ...` instead. You don't have guarantee that build host's rpm is compatible with rpmdb inside the template.